src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -46,6 +46,132 @@
             <column name="last_modified_date" type="timestamp"/>
         </createTable>
 
+<!--        Address-->
+
+        <createTable tableName="address_state">
+            <column name="id" type="bigint" >
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="state" type="varchar(20)" />
+        </createTable>
+
+        <loadData tableName="address_state" file="config/liquibase/data/address_state.csv" separator=";">
+            <column name="id" type="numeric" />
+            <column name="state" type="string" />
+        </loadData>
+
+        <createTable tableName="address">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="street_address" type="varchar(50)" />
+            <column name="unit" type="varchar(20)" />
+            <column name="city" type="varchar(20)" />
+            <column name="state" type="bigint" />
+            <column name="zip_code" type="varchar(10)" />
+            <column name="user_id" type="bigint" />
+        </createTable>
+
+        <addForeignKeyConstraint baseTableName="address" baseColumnNames="state" constraintName="state_id" referencedTableName="address_state"
+                                 referencedColumnNames="id"/>
```

Comment:
I think we need an additional foreign key constraint for the user id (unless you have that somewhere else and I am missing it)